src/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/SignedCms.cs

```diff
@@ -45,12 +46,18 @@ public SignedCms(SubjectIdentifierType signerIdentifierType, ContentInfo content
             if (contentInfo.Content == null)
                 throw new ArgumentNullException("contentInfo.Content");
 
-            // signerIdentifierType is ignored.
-            // In .NET Framework it is used for the signer type of a prompt-for-certificate signer.
-            // In .NET Core we don't support prompting.
-            //
-            // .NET Framework turned any unknown value into IssuerAndSerialNumber, so no exceptions
-            // are required, either.
+            // Normalize the subject identifier type
+            switch (signerIdentifierType)
+            {
+                case SubjectIdentifierType.NoSignature:
+                case SubjectIdentifierType.IssuerAndSerialNumber:
+                case SubjectIdentifierType.SubjectKeyIdentifier:
+                    _signerIdentifierType = signerIdentifierType;
+                    break;
+                default:
+                    _signerIdentifierType = SubjectIdentifierType.IssuerAndSerialNumber;
```

Comment:
I think I'd leave some parts of the comments which explain why we don't throw here

Reply:
It's already tested by a unit test, but on the other hand comment won't hurt either.

Reply:
Well, in this case it's "because it's what NetFX does".  But, sure, I can add a comment.